src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -726,8 +738,18 @@ private static void EmitOutput(JsonTextWriter jsonWriter, OutputSymbol outputSym
             else
             {
                 emitter.EmitProperty("value", outputSymbol.Value);
+                // emit any decorators on this output
+                if (EvaluateDecorators(
+                    outputSymbol.DeclaringOutput, 
+                    SyntaxFactory.CreateObject(Enumerable.Empty<ObjectPropertySyntax>()), 
+                    outputSymbol.Type) is ObjectSyntax body)
```

Comment:
No need for the `if` check here - `EvaluateDecorators` will always return a non-null `ObjectSyntax`, even if there are no decorators.